src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 1px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__product-image {
+    width: 160px;
+    margin: 0 3px;
+    height: 134px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $primary-color;
+    text-align: left;
+    margin: 40px 0 0;
+  }
+
+  &__product-code {
+    color: $secondary-color;
+    width: 100%;
+    margin: 4px 0 0;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    width: 100%;
+    margin-top: 16px;
```

Comment:
abo you use margin, here margin-top, preferably to use same approach everywhere
I prefer shorthand of margin, because it's easier to extend it